cmd/experimental/podtaintstolerations/hack/e2e-test.sh

```diff
@@ -19,60 +19,60 @@ set -o nounset
 set -o pipefail
 
 SOURCE_DIR="$(cd "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
-ROOT_DIR=$SOURCE_DIR/../../../..
-export KUSTOMIZE=$ROOT_DIR/bin/kustomize
-export GINKGO=$ROOT_DIR/bin/ginkgo
-export KIND=$ROOT_DIR/bin/kind
+ROOT_DIR="$SOURCE_DIR/../../../.."
+export KUSTOMIZE="$ROOT_DIR"/bin/kustomize
+export GINKGO="$ROOT_DIR"/bin/ginkgo
+export KIND="$ROOT_DIR"/bin/kind
```

Comment:
```suggestion
export KUSTOMIZE="$ROOT_DIR/bin/kustomize"
export GINKGO="$ROOT_DIR/bin/ginkgo"
export KIND="$ROOT_DIR/bin/kind"
```

Could you address this in other places as well?

Reply:
Oh! I just saw this. I'll address this in other places as well.

Reply:
Done.